### DISCUSSION TITLE
766

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#934] [Comments]
# 766

2021년 초에 합류했다가 벌써 만 3년이 다되어간다. 올해는 점점 내 역할을 팀원들에게 위임하는 한 해였고, 제품의 성적표와 회사의 매출이 연결되는 것을 증명하는 해였던것 같다. 개인 회고는 별도로 작성할 예정이다. 1. 조직 개발팀 미션 인프랩 개발팀의 미션과 가치을 작성해서 팀에 공유했다. 사내 Wiki에 먼저 기록하고 기술 블로그에 공개적으로도 공유했다. 매 주마다 진행하는 개발팀 스프린트에서 자주 하던 이야기들이 있다. 득점왕 보다는 팀을 우승시키는 팀원이 되자. 계몽보다는 전염시키자. 숙련도가 낮음을 기술의 문제로 돌리지말라. 기타 등등 이런 이야기들을 자주 하다보니 개발팀 전체가 어떤 사람이 우리팀에 어울리는지, 어떤 동료가 좋은 동료인지 추상화된 형태로 다들 이해는 하고 있는 상태였다. 다..

https://jojoldu.tistory.com/766

**Comment by oy-janghoesoo** [comment]
항상~!!! 응원합니다!!!!

**Reply by jojoldu** [reply]
감사합니다 희수님! ㅎㅎ 저도 항상 응원합니다 :)

**Comment by AppleCEO** [comment]
아침에 책은 어디에서 몇시에 읽으시는지 궁금합니다 ㅎㅎ

**Reply by jojoldu** [reply]
아침 독서는 새로 이사한 집에서 읽고 있습니다 :) 
이것 관련해서는 개인 회고에서 좀 더 자세히 말씀드릴 수 있을것 같아요 :)

**Comment by seriouskang** [comment]
다른 분들의 회고를 이렇게 흥미롭게 본 건 거의 처음인 것 같습니다.
특히나 "Cell 조직 개편" 부분은 조직원으로서 공감되는 부분이 많아서, 감탄했습니다!!

요즘 업무 관련해서 스트레스가 많아 열정이 많이 식었는데,
향로님 글을 보니 열심히 해야겠다는 생각이 드네요.

좋은 글 감사합니다!

**Comment by hyunjunspark** [comment]
안녕하세요! 글정말 잘봤습니다!! 
글을 읽다가 한가지 궁금한점이 있습니다. 
인프런 전체나 테크 조직의 경우 개개인의 인사평가를 어떤식으로 하는지  궁금하네요. 
팀워크가 중요하다면 상대평가가 어울리지 않을것 같다는 생각이들어서 어떤식으로 평가를 진행하는지 궁금합니다.
